From 8f6f4e00ec41396460a965fe97cea444ec384f02 Mon Sep 17 00:00:00 2001
From: rick venn <rv@lap.chop>
Date: Thu, 7 Feb 2019 21:06:37 +0000
Subject: [PATCH] fix gcc new -Wstringop-truncation warning

---
 matron/src/args.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/matron/src/args.c b/matron/src/args.c
index 14a1375..fe91866 100644
--- a/matron/src/args.c
+++ b/matron/src/args.c
@@ -21,10 +21,10 @@ int args_parse(int argc, char **argv)
     while( ( opt = getopt(argc, argv, "r:l:m:") ) != -1 ) {
         switch(opt) {
         case 'l':
-            strncpy(a.loc_port, optarg, ARG_BUF_SIZE);
+            strncpy(a.loc_port, optarg, ARG_BUF_SIZE-1);
             break;
         case 'r':
-            strncpy(a.rem_port, optarg, ARG_BUF_SIZE);
+            strncpy(a.rem_port, optarg, ARG_BUF_SIZE-1);
             break;
         default:
             ;;
-- 
2.19.2

