From bcd6d130e85bb1e601ef3315e8e5fa2ded3a1fbb Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Sat, 2 Mar 2019 16:11:12 +0100
Subject: [PATCH] Remove sudo from shell invocations

---
 lua/menu.lua | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/lua/menu.lua b/lua/menu.lua
index c1f889c..db60dba 100644
--- a/lua/menu.lua
+++ b/lua/menu.lua
@@ -1148,7 +1148,7 @@ end
 m.wifi.passdone = function(txt)
   if txt ~= nil then
     os.execute("~/norns/wifi.sh select \""..m.wifi.try.."\" \""..txt.."\" &")
-    os.execute("sudo systemctl stop norns-crone.service")
+    os.execute("systemctl stop norns-crone.service")
     wifi.on()
     m.wifi.countdown = 4
   end
@@ -1326,17 +1326,17 @@ m.key[pSYNC] = function(n,z)
   elseif n==3 and z==1 and m.sync.pos==0 then
     m.sync.busy = true
     menu.redraw()
-    os.execute("sudo rsync --recursive --links --verbose --update $HOME/dust/ "..m.sync.disk.."/dust; sudo sync")
+    os.execute("rsync --recursive --links --verbose --update $HOME/dust/ "..m.sync.disk.."/dust; sync")
     norns.log.post("sync to usb")
     menu.set_page(pSYSTEM)
   elseif n==3 and z==1 and m.sync.pos==1 then
     m.sync.busy = true
     menu.redraw()
-    os.execute("rsync --recursive --links --verbose --update "..m.sync.disk.."/dust/ $HOME/dust; sudo sync")
+    os.execute("rsync --recursive --links --verbose --update "..m.sync.disk.."/dust/ $HOME/dust; sync")
     norns.log.post("sync from usb")
     menu.set_page(pSYSTEM)
   elseif n==3 and z==1 and m.sync.pos==2 then
-    os.execute("sudo umount "..m.sync.disk)
+    os.execute("umount "..m.sync.disk)
     norns.log.post("usb disk ejected")
     menu.set_page(pSYSTEM)
   end
@@ -1497,7 +1497,7 @@ end
 m.init[pRESET] = function()
   m.reset.countdown = 9
   norns.script.clear()
-  os.execute("sudo systemctl restart norns-crone.service")
+  os.execute("systemctl restart norns-crone.service")
   u.time = 1
   u.count = -1
   u.callback = function()
@@ -1533,7 +1533,7 @@ m.key[pSLEEP] = function(n,z)
     norns.audio.output_level(-100)
     gain_hp(0)
     wifi.off()
-    os.execute("sleep 0.5; sudo shutdown now")
+    os.execute("sleep 0.5; shutdown now")
   end
 end
 
-- 
2.19.2

