From 7a7e116b84b2a6aaeeb28098f6f90f89e54f187a Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Sun, 19 May 2019 22:21:52 +0200
Subject: [PATCH] sc/wscript: Install SC files to system-wide Supercollider
 extension directory

---
 sc/wscript | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/sc/wscript b/sc/wscript
index fe18b0e..9534ba5 100644
--- a/sc/wscript
+++ b/sc/wscript
@@ -1,4 +1,8 @@
-top = '..'
+def set_options(opt):
+    pass
+
+def configure(conf):
+    pass

 def build_ugen(bld, target, source):
     ugen = bld.shlib(features='c cxx',
@@ -34,3 +38,7 @@ def build(bld):
     for u in ugens:
         build_ugen(bld, u['target'], u['source'])
         build_ugen_supernova(bld, u['target'], u['source'])
+
+    # Install SC files
+    bld.install_files('${PREFIX}/share/SuperCollider/Extensions/norns', bld.path.ant_glob('core/**/*'), relative_trick=True)
+    bld.install_files('${PREFIX}/share/SuperCollider/Extensions/norns', bld.path.ant_glob('engines/**/*'), relative_trick=True)
--
2.21.0
